src/components/Entry.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+const Title = styled.span`
```

Comment:
Totally fine, but no need to over-engineer stuff like this. A good rule: If the styling is unique to this component, just style the tags in the top level styled component:

```js
const StyledEntry = styled.div`
  margin: 10px;

  p {
    font-size: 30px;
  }
`
```

If you find yourself writing the same style for multiple tags across component, now we have a reason to move it into `/styles` directory.

Also, a good naming convention for the styled component is `StyledNameOfElement`. This is because later down the line you could end up writing a react `Title` component, and then you'd have a import conflict if you want to use the Component, and the Styles in the same file